base/util/src/main/java/com/netscape/cmsutil/crypto/CryptoUtil.java

```diff
@@ -2091,12 +2140,19 @@ private static PKIArchiveOptions createPKIArchiveOptionsInternal(
             SymmetricKey symKeyData,
             WrappingParams params,
             AlgorithmIdentifier aid) throws Exception {
+
+        SymmetricKey.Usage[] usages = new SymmetricKey.Usage[4];
+        usages[0] = SymmetricKey.Usage.WRAP;
+        usages[1] = SymmetricKey.Usage.UNWRAP;
+        usages[2] = SymmetricKey.Usage.ENCRYPT;
+        usages[3] = SymmetricKey.Usage.DECRYPT;
```

Comment:
Similarly, if this is a constant we might want to convert it into a class/instance variable.